docs/api.rst

```diff
@@ -9182,24 +9193,12 @@ Deleting manually tracked balances
       {
           "result": {
           "balances": [{
-                  "asset": "XMR",
```

Comment:
Removed these objects because it looks strange when items are deleted but they are still returned